genesis_core/user_api/iam/api/controllers.py

```diff
@@ -44,6 +47,33 @@ def enforce(self, rule, do_raise=False, exc=None):
         return iam.enforcer.enforce(rule, do_raise, exc)
 
 
+class ValidateMixin:
+    vm_min_length = 8
+    vm_not_contain: str = string.whitespace
+    vm_must_contain: list[str] = None  # [digits, ascii_uppercase, punctuation]
+    vm_regex: str = None
+
+    def validate(self, text):
+        if text is None:
+            return
```

Comment:
are you sure we should skip the validation, if new password (or any other validating string) is None?